src/commands/uplink/GetLmicVersion.ts

```diff
@@ -0,0 +1,125 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import BinaryBuffer from '../../BinaryBuffer.js';
+import * as bitSet from '../../utils/bitSet.js';
+import {UPLINK} from '../../constants/directions.js';
+
+
+interface ILmicStatus extends bitSet.TBooleanObject {
+    multicastSupport: boolean,
+    /* LoRaWAN Fragmented Data Block Transport  */
+    fragmentedDataSupport: boolean
+}
+
+/**
+ * GetLmicVersion command parameters
+ */
+interface IUplinkGetLmicVersionParameters {
+    /* status of supported features */
+    status: ILmicStatus,
+    version: number
+}
+
+
+const COMMAND_ID = 0x021f;
+const COMMAND_TITLE = 'GET_LMIC_VERSION';
+const COMMAND_BODY_SIZE = 2;
+
+const lmicStatusBitMask = {
```

Comment:
is it really `status`?

in the NEW_STATUS command they call it `type`

Reply:
status renamed to capabilities

Reply:
lmicStatusBitMask